contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -89,6 +94,11 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
      */
     event ProgramEnabled(Token indexed pool, bool status, uint256 remainingRewards);
 
+    /**
+     * @dev triggered when the auto trigger count is updated
+     */
+    event AutoTriggerCountUpdated(uint256 prevAutoTriggerCount, uint256 newAutoTriggerCount);
```

Comment:
@yudilevi is this necessary? It costs more gas

Reply:
the event? it's an admin function, we always add events when updating admin settings

Reply:
Oh, yeah. Resolving.